pom.xml

```diff
@@ -890,7 +890,7 @@
       <dependency>
         <groupId>commons-io</groupId>
         <artifactId>commons-io</artifactId>
-        <version>2.6</version>
+        <version>2.10.0</version>
```

Comment:
in the meantime Apache published 2.11.0 : https://commons.apache.org/proper/commons-io/changes-report.html

Reply:
Unfortunately, 2.11.0 deprecates `DirectoryWalker` which is extended in `ImageMosaicDirectoryWalker` which is a whole new can of worms we probably don't want to wait on solving.